dunwoody_disaster/views/action_selector.py

```diff
@@ -65,14 +66,27 @@ def getDefense(self) -> Item.Armor:
         return self.defense
 
     def updateUI(self):
+        attack = ""
         if self.attack:
-            self.attack_pic.setPixmap(QPixmap(self.attack.image).scaledToWidth(50))
+            attack = self.attack.image
+            if self._hidden:
+                attack = DD.ASSETS["no_texture"]
```

Comment:
@EngineeringDude123 here's another image we're going to need. Would something like [🚫](https://cdn.pixabay.com/photo/2012/04/24/12/29/no-symbol-39767_1280.png) make sense?